scalalib/src/mill/scalalib/ApeModule.scala

```diff
@@ -0,0 +1,111 @@
+package mill.scalalib
+
+import mill.api.Loose.Agg
+import mill.api.{PathRef, Result}
+import mill.define.{Target => T, _}
+
+trait ApeModule extends mill.Module with AssemblyModule {
+  def cosmoccVersion: T[String] = Task { "" }
+
+  def cosmoccZip: T[PathRef] = Task {
```

Comment:
This task should be at least `persistent`, to avoid re-downloads. Better would be to use some machine-wide cache. (https://github.com/com-lihaoyi/mill/issues/3930)

Reply:
Agree. A machine-wide cache is best here. Downloading a >400MB `cosmocc.zip` for every project is not ideal.